test/utils/dateUtils.test.ts

```diff
@@ -1,13 +1,22 @@
 import { expect } from 'chai';
-import { getWeeksBetween, toReadableDate } from '@/utils/dateUtils';
+import {
+  getEarliestDate,
+  getWeeksBetween,
+  toReadableDate,
+} from '@/utils/dateUtils';
 
 describe('dateUtils', () => {
-  describe('week calculations', () => {
+  describe('Calculations', () => {
     it('Will calculate number of weeks between two dates', () => {
       const date1 = new Date('2022-10-01');
-      const date2 = new Date('2022-10-14');
+      const date2 = new Date('2022-10-15');
       expect(getWeeksBetween(date1, date2)).to.equal(2);
```

Comment:
Når vi bruker `dayjs` til å regne ut dette, så runder man ned antall uker, så må være mer enn 14 dager for å være 2 uker. Vi diskuterte dette da vi gjorde samme endring i syfomodiaperson, og landet på at det blir mest riktig å gjøre det på denne måten: Hvis man er sykmeldt i 1,6 uker, så er ikke det 2 uker _enda_.